src/plugins/console/api_server/es_5_0/cat.js

```diff
@@ -0,0 +1,56 @@
+let _ = require("lodash");
+
+function addSimpleCat(endpoint, api, params, patterns) {
+  var url_params = {"help": "__flag__", "v": "__flag__", "bytes": ["b"]};
+  _.each(params || [], function (p) {
+    if (_.isString(p)) {
+      url_params[p] = "__flag__";
+    }
+    else {
+      var k = Object.keys(p)[0];
+      url_params[k] = p[k];
+    }
+  });
+  api.addEndpointDescription(endpoint, {
+    match: endpoint,
```

Comment:
I would have to agree with you man! No worries I was mainly asking here to try and assess the differences between this function and the one below it. I would have to definitely agree that this should be deferred to the scope of another PR.